src/slurm.jl

```diff
@@ -6,15 +6,17 @@ struct SlurmManager <: ClusterManager
     np::Integer
 end
 
+slurm_worker_arg() = `--worker=$(Distributed.init_multi(); Distributed.LPROC.cookie)`
```

Comment:
I think it is better to change `worker_arg()` instead. Because I see other managers also using it in a similar way. Maybe like: 
```
worker_arg() = `--worker=$(Distributed.init_multi(); cluster_cookie())
```

The other option is for user code to do it, e.g.:
```
using Distributed, ClusterManagers
Distributed.init_multi()
```

But modifying `worker_arg()` makes it transparent and maybe better.